c_glib/test/test-memory-pool.rb

```diff
@@ -20,6 +20,7 @@ class TestMemoryPool < Test::Unit::TestCase
 
   def setup
     @memory_pool = Arrow::MemoryPool.default
+    Arrow::ResizableBuffer.new(1)
```

Comment:
We need to assign this to a variable to guard from GC during each test.

```suggestion
    # Our tests assume that some memory allocated.
    @buffer = Arrow::ResizableBuffer.new(1)
```

Reply:
Thanks! Fixed.